modules/concepts/hooks/list-of-hooks.md

```diff
@@ -2620,14 +2620,14 @@ validateCustomerFormFields
 : 
     Located in: /classes/form/CustomerForm.php
 
-action<KpiIdentifier>KpiRowModifier
+action&lt;KpiIdentifier>KpiRowModifier
 : 
     Available since: {{< minver v="1.7.6" >}}
 
     This hook allow to alter the list of Kpis used in a Kpi row.
     This hook is called just before the validation and the  building of the KpiRow.
```

Comment:
```suggestion
    This hook is called just before the validation and the building of the KpiRow.
```
It's not really related, but as I see it in the same time, :)

Reply:
Thanks @PrestaEdit